src/applications/rated-disabilities/tests/fixtures/feature-toggles.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I kept this around because using the intercept allows us to lower the amount of time it takes for the tests to run